src/main/java/nl/hsac/fitnesse/util/iban/IbanUtil.java

```diff
@@ -0,0 +1,129 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+
+/**
+ * To select a country and parse the iban request to the proper class
+ */
+public class IbanUtil {
+    private final RandomUtil RANDOM_UTIL = new RandomUtil();
+
+
+    public String generateIban(String country, String bankCode) {
+        String iban;
+        if (country.equals("")) {
+            country = RANDOM_UTIL.randomElement(countryCodes);
+        }
+
+        switch (country) {
+            case "BE":
+                BEIban beIban = new BEIban();
```

Comment:
I need to read up on reflection as I have no idea how to do this in a nicer way.
Also, I will ask Marit over lunch, she probably knows.